main.go

```diff
@@ -91,7 +93,8 @@ func init() {
 	flag.Int64Var(&schemaSampleSize, "schema-sample-size", int64(100000), "schema-sample-size: the number of rows to use for inferring schema (only for DynamoDB)")
 	flag.BoolVar(&verbose, "v", false, "verbose: print additional output")
 	flag.BoolVar(&schemaOnly, "schema-only", false, "schema-only: mode for schema only conversion to Spanner")
-
+	flag.BoolVar(&dataOnly, "data-only", false, "data-only: mode for data only conversion to Spanner")
+	flag.StringVar(&sessionJSON, "session", "", "session: session file generated through schema-only or web mode")
```

Comment:
This comment is a little confusing: not sure if it is an input or name of file we will generate.
Maybe drop the comment about web mode since we don't have that yet?

Suggest rephrasing as:
"session: specifies the file we restore session state from (used in schema-only to provide schema and mapping configuration)"

Reply:
Sure. Done.